### PR TITLE
Add update outfit option, sort outfit menus

### PR DESCRIPTION
**Pull request by Mkeefeus**
Adds an update outfit option to the clothing store menu so you can change it without needing to delete the existing outfit. Also sorts the update and change outfit menus alphabetically, as I don't think there was any sorting before

### CODE REVIEW

**Comment by TheiLLeniumStudios** [comment]
@Mkeefeus Thank you for the PR. Could you please sort the outfits when displaying them in the Delete and Change menu? I'd like everything to be consistent

File: server/server.lua [hunk 1]
```diff
@@ -214,6 +214,34 @@ RegisterNetEvent("illenium-appearance:server:saveOutfit", function(name, model,
...
+                outfit.model = model
+                outfit.components = components
+                outfit.props = props
+                outfitName = outfit.name
+                break
+            end
+        end
+        lib.notify(src, {
+            title = "Success",
+            description = "Outfit " .. outfitName .. " has been saved",
```
Comment (TheiLLeniumStudios) [inline server/server.lua:238]:
```suggestion
            description = "Outfit " .. outfitName .. " has been updated",

```

**Comment by Mkeefeus** [comment]
> @Mkeefeus Thank you for the PR. Could you please sort the outfits when displaying them in the Delete menu as part of the same PR? I'd like everything to be consistent

Delete menu should now be sorted as well

**Comment by TheiLLeniumStudios** [comment]
Thank you @Mkeefeus . LGTM

**Review by GodyFromDiscord — COMMENTED** [review]
The updateOutfit event has an = rather than a - by illenium-appearance on the new events

**Comment by Mkeefeus** [comment]
> The updateOutfit event has an = rather than a - by illenium-appearance on the new events

Whoops. its like that for both the Register and the Trigger so it still works, but should be changed for sure

**Pull request merged**